Source/Core/UICommon/AutoUpdate.h

```diff
@@ -54,9 +54,9 @@ class AutoUpdateChecker
     NO_RESTART_AFTER_UPDATE = 0,
     RESTART_AFTER_UPDATE,
   };
-  bool TriggerUpdate(const NewVersionInformation& info, RestartMode restart_mode) const;
+  bool TriggerUpdate(const NewVersionInformation& info, const RestartMode restart_mode) const;
 
 protected:
-  virtual void OnUpdateAvailable(const NewVersionInformation& info) = 0;
+  virtual void OnUpdateAvailable(const NewVersionInformation& info) const = 0;
   virtual void OnErrorOccurred(const CheckError error) const = 0;
```

Comment:
ditto for error

Reply:
I asked for this one. I presume your reasoning is that it's passed by value so the caller doesn't need to worry about the const? 

That makes sense to me.

Reply:
Yeah. const prevents `error` from being mutated, but what's going to mutate its value in the declaration? Nothing, so it's just noise in this case.